Assets/Plugins/Source/Editor/EpicOnlineServicesConfigEditor.cs

```diff
@@ -31,29 +31,12 @@
 
 namespace PlayEveryWare.EpicOnlineServices
 {
-
-    public class EpicOnlineServicesConfigEditor : EditorWindow
+    [Serializable]
+    public class EpicOnlineServicesConfigEditor : EOSEditorWindow
     {
-        static Regex EncryptionKeyRegex;
-
-        static EpicOnlineServicesConfigEditor()
-        {
-            EncryptionKeyRegex = new Regex("[^0-9a-fA-F]");
-        }
-
-        public static void AddPlatformSpecificConfigEditor(IPlatformSpecificConfigEditor platformSpecificConfigEditor)
-        {
-            if (platformSpecificConfigEditors == null)
-            {
-                platformSpecificConfigEditors = new List<IPlatformSpecificConfigEditor>();
-            }
-            platformSpecificConfigEditors.Add(platformSpecificConfigEditor);
-        }
-
-        private static string IntegratedPlatformConfigFilenameForSteam = "eos_steam_config.json";
-
-        static List<IPlatformSpecificConfigEditor> platformSpecificConfigEditors;
```

Comment:
It was necessary to remove these static members / methods in order to make our editor window play nice with Unity.